### DISCUSSION TITLE
Is there any simple way to attach all partitions at once?

### DISCUSSION THREAD

**Discussion by HwiLu** [ClickHouse/ClickHouse#60261] [Q&A] [answered]
When using `alter table attach partition` to modify the MergeTree  table   to ReplicatedMergeTree table , Is there any simple way to attach all partitions at once?

**Comment by UnamedRus** [comment, marked as answer]
No, https://github.com/ClickHouse/ClickHouse/issues/59376

**Reply by HwiLu** [reply]
Thank you, it seems I am not the only one who has this need
